examples/3d/anti_aliasing.rs

```diff
@@ -1,5 +1,9 @@
 //! This example compares MSAA (Multi-Sample Anti-aliasing), FXAA (Fast Approximate Anti-aliasing), and TAA (Temporal Anti-aliasing).
 
+// This lint usually gives bad advice in the context of Bevy -- hiding complex queries behind
+// type alias tends to obfuscate code while offering no improvement in code cleanliness.
```

Comment:
```suggestion
// This lint usually gives bad advice in the context of Bevy -- hiding complex queries behind
// type aliases tends to obfuscate code while offering no improvement in code cleanliness.
```

I had a grammatical error in my original suggestion.

Reply:
I should've noticed that. Fixed and force pushed.